internal/plugin/plugin.go

```diff
@@ -75,7 +75,7 @@ func Walk(dir string, f func(plugin *Plugin)) error {
 		if p1.Name != p2.Name {
 			return p1.Name < p2.Name
 		}
-		return semver.Compare(p1.Version, p2.Version) < 0
+		return semver.Compare(p1.PluginVersion, p2.PluginVersion) < 0
```

Comment:
The ExternalConfig Version is set to `v1` everywhere - we want to use the `PluginVersion` everywhere we used `Version` before.